### PR TITLE
Impl `into` for `B512`

### LINKED ISSUE #1792: Add into() for B512
As per the comments:
>pub trait From {
    fn from(h: b256, l: b256) -> Self;
} {
    // @todo add into() when tuples land, as it would probably return 2 b256 values
    // fn into() {...}
}

### PR DESCRIPTION
**Pull request by nfurfaro**
All in the title. 

closes #1792 
ref #1795

### CODE REVIEW

File: sway-lib-std/src/storage.sw [hunk 1]
```diff
@@ -92,7 +92,8 @@ pub fn get<T>(key: b256) -> T {
     }
 }
 
-pub struct StorageMap<K, V> { }
+pub struct StorageMap<K, V> {
```
Comment (adlerjohn) [inline sway-lib-std/src/storage.sw:95]: Remove drive-by formatting

**Review by otrho — APPROVED** [review]
